docs/sql-data-sources-json.md

```diff
@@ -114,62 +114,62 @@ Data source options of JSON can be set via:
   <tr>
     <!-- TODO(SPARK-35433): Add timeZone to Data Source Option for CSV, too. -->
     <td><code>timeZone</code></td>
-    <td>None</td>
+    <td>The SQL config <code>spark.sql.session.timeZone</code></td>
```

Comment:
```suggestion
    <td>(value of <code>spark.sql.session.timeZone</code> configuration)</td>
```

Reply:
to match with https://spark.apache.org/docs/latest/configuration.html

Reply:
can you fix other instances too? e.g. in CSV as well https://github.com/apache/spark/blob/73fd6de9a18e8b550fd9afbcf9c87efa598fd76e/docs/sql-data-sources-csv.md

Reply:
and parquet https://github.com/apache/spark/blob/73fd6de9a18e8b550fd9afbcf9c87efa598fd76e/docs/sql-data-sources-parquet.md

Reply:
and avro https://github.com/apache/spark/blob/73fd6de9a18e8b550fd9afbcf9c87efa598fd76e/docs/sql-data-sources-avro.md